src/main/java/mate/jdbc/dao/CarDaoImpl.java

```diff
@@ -0,0 +1,237 @@
+package mate.jdbc.dao;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import mate.jdbc.exception.DataProcessingException;
+import mate.jdbc.lib.Dao;
+import mate.jdbc.model.Car;
+import mate.jdbc.model.Driver;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.util.ConnectionUtil;
+
+@Dao
+public class CarDaoImpl implements CarDao {
+    private static final String CAR_ID_COLUMN = "id";
+    private static final String CAR_MODEL_COLUMN = "model";
+    private static final String CAR_MANUFACTURER_ID_COLUMN = "manufacturer_id";
+    private static final String MANUFACTURER_NAME_COLUMN = "name";
+    private static final String MANUFACTURER_COUNTRY_COLUMN = "country";
+    private static final String DRIVER_ID_COLUMN = "id";
+    private static final String DRIVER_NAME_COLUMN = "name";
+    private static final String DRIVER_LICENSE_NUMBER_COLUMN = "license_number";
+
+    @Override
+    public Car create(Car car) {
+        String createCarRequest = "INSERT INTO cars (model, manufacturer_id) VALUES (?, ?);";
```

Comment:
```suggestion
        String createCarQuery= "INSERT INTO cars (model, manufacturer_id) VALUES (?, ?);";
```
likewise in the following cases